clang/test/SemaSYCL/array-num-elements-exceeded.cpp

```diff
@@ -0,0 +1,18 @@
+// RUN: %clang_cc1 -I %S/Inputs -fsycl -fsycl-is-device -triple spir-unknown -verify -fsyntax-only %s
+// RUN: %clang_cc1 -I %S/Inputs -fsycl -fsycl-is-device -triple spir64-unknown -verify -fsyntax-only %s
+#include "big_constant.h"
```

Comment:
Is there value to having this file vs something line:
double big[67000] = { 1., 2., 3. } ?

Reply:
Good suggestion.
Declaring `double big[67000] = { 1., 2., 3. }`, will have `Value->getArrayInitializedElts() == 3` ( the rest of the array elements get set to 0) and the size of the array will be 67000.
Removed big_constant.h and updated test.